src/components/Uploader.vue

```diff
@@ -21,25 +21,26 @@
 
       <template v-slot:list="scope">
         <q-list separator>
-          <q-item v-if="hasAPIValue && !scope.isUploading" class="q-pa-none">
-            <q-item-section avatar top>
-              <q-avatar v-if="value" rounded>
-                <img :src="value">
-              </q-avatar>
-
-              <q-avatar v-else color="grey-3" icon="o_attach_file" rounded :text-color="isFileFailed(file) ? 'negative' : 'primary'" />
```

Comment:
Por que removeu o ícone? E quando o upload não for de imagem?

Reply:
criei uma função para verificar o `type` e adicionei o ícone.